docs/core_data_structure.md

```diff
@@ -1,6 +1,186 @@
 (data-structure)=
 # Core data structures
 
-The whiteboard image below captures the planned core data structure of the `Survey` object that is central to Echopop. Most of the structure is in place but we will fully implement them very soon (tracked in issue [#169](https://github.com/OSOceanAcoustics/echopop/issues/169)).
+Input data, intermediate variables and calculations, and results are stored within several attributes associated with the `Survey`-class object. Information on how these attributes are used in the general `echopop` workflow can be found in greater detail [within the example notebook](echopop-example-workflow). 
```

Comment:
This is ok for now, but I think we should have a diagram or table on the top of this page as a quick reference. I'll add an issue to track this.

Reply:
Oh I had missed what you have as a TOC since it didn't get rendered in GH, but it got rendered in RTD -- awesome! I would just suggest some edits here then!
```suggestion
Input data, intermediate variables and calculations, and results are stored within several attributes associated with the `Survey`-class object. The attributes and stored data are detailed in the sections below. See [the example notebook](echopop-example-workflow) for information on how these attributes are used in the general `echopop` workflow. 
```